contracts/OpenStream.sol

```diff
@@ -103,6 +103,12 @@ contract OpenStream is ReentrancyGuard, IOpenStream {
         isClaimable = _isClaimable;
     }
 
+    ///@dev it setting a new rate for this contract(instance open stream).
+    /// Can call only `admin` or `payer`.
+    function setNewRate(uint256 _rate) public {
+        if (msg.sender == admin || msg.sender == payer) rate = _rate;
+    }
```

Comment:
why not to use modifier?

Reply:
Because this is the function itself that sets a new value for `rate`.
And before that, there is a check that only two users (`admin ` and `payer`) can set a new value for `rate`